test/dynamo/test_higher_order_ops.py

```diff
@@ -2665,6 +2666,286 @@ def _compile_check(self, fn, inputs, fullgraph=True, graph_idx=0):
         wrapped_gm = backend.graphs[graph_idx]
         return wrapped_gm
 
+    @config.patch(capture_func_transforms=True)
```

Comment:
Can we change capture_func_transforms to True by default after this PR?

Reply:
Yes! I'll send a PR shortly